apps/mobile/src/graphql/queries/groupDetailsQueryMaker.js

```diff
@@ -9,13 +9,13 @@ import groupWidgetsFieldsFragment from 'graphql/fragments/groupWidgetsFieldsFrag
 
 // Note: The previous defaults args for fetchGroupDetails were:
 // withExtensions = true,
-// withWidgets = false,
+// withContextWidgets = false,
 // withTopics = true,
 // withJoinQuestions = true,
 // withPrerequisites = true
 export default function groupDetailsQueryMaker ({
   withExtensions = false,
-  withWidgets = false,
+  withContextWidgets = false,
```

Comment:
consider making this `withWidgets` and removing the word context from the fragment names as well. I feel like Context is implied, and is not part of the graphql schema naming of a widget.